[2025-07-23T23:20:22.819+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: TechAndUrbanGrowth.ingestion_air_quality_data scheduled__2025-07-22T00:00:00+00:00 [queued]>
[2025-07-23T23:20:22.978+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: TechAndUrbanGrowth.ingestion_air_quality_data scheduled__2025-07-22T00:00:00+00:00 [queued]>
[2025-07-23T23:20:22.984+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-23T23:20:23.115+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ingestion_air_quality_data> on 2025-07-22 00:00:00+00:00
[2025-07-23T23:20:23.160+0000] {standard_task_runner.py:60} INFO - Started process 186 to run task
[2025-07-23T23:20:23.357+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'TechAndUrbanGrowth', 'ingestion_air_quality_data', 'scheduled__2025-07-22T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp0yk8jsjt']
[2025-07-23T23:20:23.410+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask ingestion_air_quality_data
[2025-07-23T23:20:24.030+0000] {task_command.py:423} INFO - Running <TaskInstance: TechAndUrbanGrowth.ingestion_air_quality_data scheduled__2025-07-22T00:00:00+00:00 [running]> on host 1222bc5070ab
[2025-07-23T23:20:24.818+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='TechAndUrbanGrowth' AIRFLOW_CTX_TASK_ID='ingestion_air_quality_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-22T00:00:00+00:00'
[2025-07-23T23:20:56.983+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Buenos Aires, ARG: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:21:27.552+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para St. John's, ATG: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:21:58.322+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Nassau, BHS: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:22:29.003+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Bridgetown, BRB: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:22:59.571+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Belmopan, BLZ: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:23:30.129+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Sucre, BOL: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:24:00.601+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Brasilia, BRA: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:24:31.180+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Ottawa, CAN: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:25:01.784+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Santiago, CHL: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:25:32.415+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Bogotá, COL: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:26:03.090+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para San José, CRI: mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68816e38d16b78d6feac4b48, topology_type: Unknown, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('mongodb:27017: [Errno -2] Name or service not known (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[2025-07-23T23:26:03.342+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2025-07-23T23:26:03.639+0000] {process_utils.py:131} INFO - Sending 15 to group 186. PIDs of all processes in the group: [186]
[2025-07-23T23:26:03.672+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 186
[2025-07-23T23:26:03.720+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-07-23T23:26:03.790+0000] {ingestion_1.py:46} ERROR - ❌ Error extrayendo calidad aire para Havana, CUB: Task received SIGTERM signal
